src/core/lib/debug/stats.cc

```diff
@@ -150,7 +150,7 @@ std::string grpc_stats_data_as_json(const grpc_stats_data* data) {
   parts.push_back("{");
   for (size_t i = 0; i < GRPC_STATS_COUNTER_COUNT; i++) {
     parts.push_back(absl::StrFormat(
-        "\"%s\": %" PRIdPTR, grpc_stats_counter_name[i], data->counters[i]));
+        "\"%s\": %" PRIdPTR ",", grpc_stats_counter_name[i], data->counters[i]));
```

Comment:
Instead of adding in the commas in each individual element here, I suggest letting `absl::StrJoin()` do this for us.  How about something like this:

```
std::vector<std::string> parts;
for (size_t i = 0; i < GRPC_STATS_COUNTER_COUNT; i++) {
  parts.push_back(absl::StrFormat(
      "\"%s\": %" PRIdPTR, grpc_stats_counter_name[i], data->counters[i]));
}
for (size_t i = 0; i < GRPC_STATS_HISTOGRAM_COUNT; i++) {
  std::vector<std::string> bucket_fields;
  // ...add entries to bucket_fields...
  parts.push_back(absl::StrFormat(
      "\"%s\": [%s]", grpc_stats_histogram_name[i],
      absl::StrJoin(bucket_fields, ", ")));
}
return absl::StrCat("{", absl::StrJoin(parts, ", "), "}");
```

Reply:
TBH, it looks a bit more confusing to me. But you believe it's better and it still works - fine by me.